tests/SpecTests/crud/aggregate-merge.json

```diff
@@ -0,0 +1,415 @@
+{
+  "runOn": [
+    {
+      "minServerVersion": "4.2.0"
```

Comment:
https://github.com/mongodb/specifications/commit/efe9319a2b8b046d44fcea9661b9e18fdb109031 was merged, so you can update this here or in #644 as you wish.

Reply:
Just saw your comment in https://github.com/mongodb/mongo-php-library/pull/644#discussion_r306689346. I suppose this is the PR to update it then.